plugins/altos/fu-altos-device.c

```diff
@@ -429,12 +429,21 @@ fu_altos_device_write_page (FuAltosDevice *device,
 	return TRUE;
 }
 
+static void
+fu_altos_device_set_progress (FuAltosDevice *device, goffset current, goffset total)
+{
+	gdouble percentage = -1.f;
+	if (total > 0)
+		percentage = (100.f * (gdouble) current) / (gdouble) total;
+	g_debug ("written %" G_GOFFSET_FORMAT "/%" G_GOFFSET_FORMAT " bytes [%.1f%%]",
```

Comment:
Maybe this should be hidden in a "plugin" level debug like introduced previously